config/environments/production.rb

```diff
@@ -76,6 +71,7 @@
 
   config.lograge.enabled = true
   config.lograge.base_controller_class = 'ActionController::API'
+  config.lograge.ignore_actions = ['OkComputer::OkComputerController#show']
```

Comment:
Do we need to see each GET request Nagios sends to the OkComputer controller every minute?